packages/compass-e2e-tests/index.ts

```diff
@@ -45,11 +43,7 @@ let metricsClient: MongoClient;
 
 const FIRST_TEST = 'tests/time-to-first-query.test.ts';
 
-async function setup() {
-  if (process.env.COMPASS_E2E_DISABLE_KEYCHAIN_USAGE !== 'true') {
-    await keychain.activate();
```

Comment:
The problem before, why we disabled the tests on macos in the first place, was that the default keychain was locked and creating an unlocked temporary one was messing with other processess on the machine, so we stopped doing this.

Electron `safeStorage` also uses keychain, the difference with keytar is that instead of multiple keys, we just store one in the keychain, so to re-enable e2e tests on macos, I would assume we'd still need those conditions to be met and I don't see anything in this PR unlocking the keychain on macos machine. So I'm wondering what in this PR actually allows us to re-enable the tests? 🤔

Reply:
With `use-mock-keychain`, for macOS, the Chromium does not trigger keychain and uses mocked keychain. Hence does not require us to unlock it. So I removed all the unlocking code for macOS.